Assets/Scripts/Gameplay/UI/IPUIMediator.cs

```diff
@@ -171,13 +172,30 @@ public void CancelConnectingWindow()
         }
 
         /// <summary>
-        /// Sanitize user port InputField box allowing only alphanumerics and '.'
+        /// Sanitize user IP address InputField box allowing only numbers and '.'
         /// </summary>
         /// <param name="dirtyString"> string to sanitize. </param>
         /// <returns> Sanitized text string. </returns>
-        public static string Sanitize(string dirtyString)
+        public static string SanitizeIP(string dirtyString)
```

Comment:
I'm curious in general, what do we gain by doing that replace ourselves? Why not rely on UTP's tryParse as a whole?

Reply:
TryParse doesn't replace, it is only a validation check. We don't need to replace either and we could decide to only keep the validation, but I think it is a nice quality of life feature that wouldn't hurt to be kept in

Reply:
oh wait, it was there because of the zoom copy paste issues we had, where copy pasting strings from zoom introduced bad invisible characters. Might be worth writing this down in a comment somewhere

Reply:
Good point! I added this explanation to the method header